### PR TITLE
Updating plot_track from gliderpy

### PR DESCRIPTION
**Pull request by FloraSauerbronn**
Adding new function plot_track to plotting.py, decorating with pandas-flavor and adding pytest .

### CODE REVIEW

**Comment by ocefpaf** [comment]
There are conflicts in this PR. You probably need to update your main branch or start new PRs directly from upstream/main. We can go over this in today's meeting.

**Comment by ocefpaf** [comment]
pre-commit.ci autofix

File: tests/test_plotting.py [hunk 1]
```diff
@@ -5,11 +5,25 @@
...
+    # Generate the plot
+    fig, ax = plot_track(df)
+
+    # Return the figure for pytest-mpl to compare
+    return fig
+
+
 @pytest.mark.mpl_image_compare(baseline_dir=root.joinpath("baseline/"))
 def test_plot_transect():
     glider_grab = GliderDataFetcher()
```
Comment (FloraSauerbronn) [inline tests/test_plotting.py:29]:
```suggestion
    """Image comparison test for plot_transect"""
    glider_grab = GliderDataFetcher()
```

Comment (FloraSauerbronn) [reply tests/test_plotting.py:29]: Added the docstring for the test in plot_transect.

**Pull request merged**